proto/metadata.proto

```diff
@@ -35,6 +35,7 @@ message Metadata {
     TESTBED_DUT_ATE_2LINKS = 3;
     TESTBED_DUT_ATE_4LINKS = 4;
     TESTBED_DUT_ATE_9LINKS = 5;
```

Comment:
Is there technically still a need for the 9LINKS or do all the 9LINK testbeds involve lags now?

Reply:
all of the 9 links got migrated to lags, so the enum is not currently in use. would you prefer I leave it as is or remove it?

Reply:
Um, I think remove. The fewer things we are maintaining, the better.

Reply:
done